src/NServiceBus.Testing.Fakes/NServiceBus.Testing.Fakes.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=fakes/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
@andreasohlund this is specific to this project, it doesn't belong in the global standards.